src/elm/Pages/Deployments/View.elm

```diff
@@ -64,56 +66,63 @@ addForm deploymentModel =
 
 {-| viewPreview : renders single deployment item preview
 -}
-viewPreview : Org -> Repo -> Deployment -> Html msg
-viewPreview org repo deployment =
+viewPreview : Posix -> Zone -> Org -> Repo -> Deployment -> Html msg
+viewPreview now zone org repo deployment =
```

Comment:
i believe the style and code for this function has been updated, and this PR is reverting some of that. 

see the icon and the content spacing

Reply:
updated when merging in master